README.md

```diff
@@ -288,6 +292,10 @@ Visit the website to get latest updates: [awesome-chatgpt-api.top](https://aweso
 
     An Android application that helps you stay in touch by scheduling and automating SMS text messages (and calls and email) while leveraging ChatGPT for message generation.
 
+- [WristAssist](https://github.com/DevEmperor/WristAssist)
```

Comment:
There is no need for one project to appear twice in the list. This part should be removed.

Reply:
Done! :)